src/data/map.js

```diff
@@ -481,6 +486,22 @@ const getPokestops = async (minLat, maxLat, minLon, maxLon, updated = 0, showPok
             excludePokemonSQL = sqlExcludeCreate;
```

Comment:
Either remove global energy filter and change line 447 and 449, or make the global filter work (and possibly add global Pokemon encounter reward filter too?).